tests/test_runlib.py

```diff
@@ -469,7 +469,8 @@ def test_record_follow_symlinked_directories(self):
         # Test that all files were recorded including files in linked subdir ...
         self.assertListEqual(
             sorted(list(artifacts_dict.keys())),
-            sorted(self.full_file_path_list + [pair[1] for pair in link_pairs]),
+            sorted(self.full_file_path_list +
+                   [pair[1] for pair in link_pairs]),
```

Comment:
This seems unrelated.

Reply:
Think it was because I was using `autopep8` and not `black`.